pvr.hts/resources/settings.xml

```diff
@@ -10,16 +10,20 @@
 	<setting id="response_timeout" type="enum" label="30007" values="1|2|3|4|5|6|7|8|9|10|11|12|13|14|15|16|17|18|19|20|21|22|23|24|25|26|27|28|29|30|31|32|33|34|35|36|37|38|39|40|41|42|43|44|45|46|47|48|49|50|51|52|53|54|55|56|57|58|59|60" default="4" />
   </category>
 
-  <category label="30100">
+  <category label="30008">
+    <setting label="30050" type="lsep"/>
+	<setting id="autorec_approxtime" type="bool" label="30051" default="true"/>
+	<setting id="autorec_maxdiff" enable="eq(-1,true)"  type="slider" option="int" range="0,5,120" label="30052" default="15" />
+  
+    <setting label="30400" type="lsep"/>
+	<setting id="total_tuners" type="enum" label="30401" values="1|2|3|4|5|6|7|8|9|10" default="0" />
```

Comment:
I just want to prevent that the user can input wrong (like 0) or unreal numbers (like 995442).

Actually the number of tuners is not elated to the number of subscriptions. (1 tuner can have multiple subscriptions). 

I just want to make this setting somehow fail save, 
i.e. if the user enters 1000 for example, everything seems fine to him until he starts zapping, the backend will be stressed and the network will be overloaded causing the tv stream to stall.
And the user might create a bug report for this...

10 subscriptions of +- 10Mbit per channel = 100Mbit.
Assuming that most users have at least a Mbit connection, this should be fail safe..